docs/src/examples/connectivity/index.mdx

```diff
@@ -46,7 +46,7 @@ By doing so, we can make use of reactions to respond to changes as we shall soon
 ## Setting up the reaction
 
 To have our user interface inform users of changes about the state of their connection, we'll need to configure a reaction.
-We'll need to '`StatefulWidget` so that the reaction can be configured within the `initState` method and disposed of when the state itself is disposed. The complete code is as follows
+We'll need to use a `StatefulWidget` so that the reaction can be configured within the `initState` method and disposed of when the state itself is going to be disposed. The complete code is as follows
```

Comment:
Can reword "disposed of" to just "disposed"

Reply:
This comment is against an outdated version. Have you looked at the most recent version? It's broken up into two sentences now. Or did you see that and now suggesting I go back to what it was before and simply remove "of"?

If I'm not mistaken, my PR should be open for edits that you could make the edits as well. Feel free to do so if you wish :)

Reply:
I think the sentence is good for now :-). Thanks for writing up the example and doc so promptly!